cypress/support/fragments/requests/newRequest.js

```diff
@@ -32,6 +33,7 @@ export default {
     cy.do(itemBarcodeInput.fillIn(newRequest.itemBarcode));
     cy.intercept('/circulation/loans?*').as('getLoans');
     cy.do(enterItemBarcodeButton.click());
+    cy.do(selectRequestType.choose(newRequest.requestType));
```

Comment:
this method is used not only in this test so this change could crash test where newRequest.requestType is not specified. 
if you want to keep it than please go to hold-slip.cy.js and also add "requestType" property
![image](https://user-images.githubusercontent.com/99860864/216340024-52a49f14-a031-4126-93ec-0133aae79a5a.png)

Reply:
fixed